.github/dependabot.yml

```diff
@@ -0,0 +1,15 @@
+---
+version: 2
+updates:
+  - package-ecosystem: "maven"
+    directory: "/"
+    schedule:
+      interval: daily
+    allow:
```

Comment:
Do you want this section? This is emissary, so `gov.nsa.emissary:emissary` won't do anything? I didn't see any refs to `gov.nsa.raft` as well.